config/initializers/feature_toggle.rb

```diff
@@ -0,0 +1,9 @@
+class FeatureToggle
+  def self.needs_question_enabled?
```

Comment:
I think `needs_question_enabled` and `show_schemes_button` are a single feature flag not two - in that they are part of schemes which all needs to be on or off together

Reply:
My thinking was I wanted to give us flexibility to enable/disable specific parts of the feature if desired - happy to change if this isn't desired 😄

Reply:
Yeah I don't think it is, it's pretty much all or nothing for schemes.